spec/features/editing_file_attachments/replace_file_attachment_file_spec.rb

```diff
@@ -27,7 +27,7 @@ def when_i_click_to_insert_an_attachment
     visit edit_document_path(@edition.document)
     find("markdown-toolbar details").click
     click_on "Attachment"
-    expect(page).to have_content("0 Bytes")
+    expect(page).to have_content(@attachment_revision.blob_revision.byte_size)
```

Comment:
I'd suggest using a bytes term here since it can be a more realistic end-to-end test when not relying on computed values (since we don't know their validity).

We have a precedent for using size as part of the naming of fixtures which makes this easier. For example 13kb-1-page-attachment.pdf

Reply:
Makes sense, I suppose that's at the compromise of having to changing the values manually if the files change.